packages/@sanity/desk-tool/src/panes/documentPane/statusBar/sparkline/documentSparkline.tsx

```diff
@@ -75,6 +55,14 @@ export function DocumentSparkline(props: DocumentSparklineProps) {
     [reviewChangesButtonWidth]
   )
 
+  // Only transition between subsequent state, not the initial
+  const [transition, setTransition] = useState(false)
+  useEffect(() => {
+    if (!transition && loaded) {
+      requestAnimationFrame(() => requestAnimationFrame(() => setTransition(true)))
+    }
```

Comment:
a bit unclear to me why we have two nested rafs here - is it intentional? if so, why is it needed? A short comment to clarify would be good.

also, should it be a return callback with cancelAnimationFrame here? If not, add a comment to explain why it's not needed?